### ISSUE TITLE
Any way to offer either monthly or yearly subscription?

### ISSUE THREAD

**Issue by jellohouse** [Glench/ExtPay#107]
I would like to offer my customers the option to pay for example either $4.99 / month or $49 / year.

Is there any way i can do this?

I thought of adding a second extension in the ExtPay dashboard for yearly... but then i would have to handle both cases in my extension. Would be great if it could be built in feature!

**Comment by FlashInThePan** [comment]
May be wrong but perhaps there is a possibility to create different extension codes in Ext Pay website for different plans and simply use the the appropriate code in the relevant part of your extension. Think we need Glen's input here though.

**Comment by burkybang** [comment]
Yep, that's a way you can do it, but it's not ideal. See here: https://github.com/Glench/ExtPay/discussions/40
